front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,188 @@
+        <input type="submit" value="Save Platform" />
+      </form>
+    </div>
+  )
+}
+
+/*the component displays and stores general info about the show*/
+
+const Description = ({ genre, description, totalEpisodes, isMovieN }) => {
+  const refDescription = useRef();
+  const refIsMovie = useRef();
+  const refGenre = useRef();
+  const refTotalEpisodes = useRef();
+  let movieV = { isMovieN };
+  if (movieV) {
+    movieV = "Yes";
+  } else {
+    movieV = "No";
+  }
+  const saveMovieData = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div class="description">
+      <br /><br />
+      <label for="genre">Genre:  </label>
+      <span id="genre" ref={refGenre}>{genre}</span><br /><br />
+      <label for="description">Description: </label>
+      <textarea id="description" value={description} ref={refDescription} readonly /><br /><br />
+      <label for="totalEpisodes">Total Episodes: </label>
+      <span id="totalEpisodes" ref={refTotalEpisodes}>{totalEpisodes}</span><br /><br />
+      <label for="movie">Is it a movie? </label>
+      <span id="movie" value={movieV} ref={refIsMovie}>{movieV}</span><br /><br />
+    </div>
+  )
+}
+
+/*the component calls other components and displays comprehensive info about the show; 
+values passed as parameters to the main components are generated by the mock data API */
+
+const IndividualShow = (props) => {
+
+  const refTitle = useRef();
+  const refCover = useRef();
+  const returnToShows = (() => {
+    window.location.href = '/my-shows/'
+  });
+  const addToInProgress = (() => {
+    window.location.href = '/my-shows/'
+  });
+
+  const addToWatched = (() => {
+    window.location.href = '/my-shows/'
+  });
+
+  let [show, setShow] = useState([]);
+
+  useEffect(() => {
+    //temporary variable to be replaced
+    let showInfo = [];
+    let showId = "54";
+    //axios.get(`https://my.api.mockaroo.com/shows/${show.id}.json?key=3010e030`)
+    axios.get(`https://my.api.mockaroo.com/shows/${showId}.json?key=27e46d40`)
+      .then((response) => {
+        showInfo.push(response.data);
+        console.log(showInfo);
+        //console.log(show);
+        setShow(showInfo);
+      })
+      .catch((err) => {
+        console.log("We likely reached Mockaroo's request limit...");
+        console.log(err);
+        showInfo.push(mockShowAPI[showId]);
+      })
+  }, [])
+
+
+  return (
+    <>
+      <Header />
+
+      <div class="showContent">
+        <fieldset class="main">
+          <div class="showDetails">
+            <fieldset >
+              {show.map(s => ( //display name of the show
+                <h3 id="title" value={s.name} ref={refTitle}>{s.name}</h3>
+              ))}
+
+              <button onClick={(e) => returnToShows()}>
+                Return to Shows
+            </button>
+
+              <div class="btnProgressContainer">
+                <button class="btnProgress" onClick={(e) => addToInProgress()}>
```

Comment:
For all buttons, I'd suggest using the buttons found on the Profile in the master branch.